src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.TestFramework/KeyVaultTestFixture.cs

```diff
@@ -159,12 +169,39 @@ private bool FromConfiguration()
             return result.AccessToken;
         }
 
+        public async Task<string> GetUserAccessToken(string authority, string resource, string scope)
+        {
+            string clientId = TestConfigurationManager.TryGetEnvironmentOrAppSetting("NativeClientId");
+            var context = new AuthenticationContext(authority, tokenCache);
```

Comment:
I generated a nuget and consumed in our Sample project, everything works fine, even the new APIs